ranger-drove/src/test/java/io/appform/ranger/drove/servicefinderhub/DroveServiceDataSourceTest.java

```diff
@@ -103,7 +105,7 @@ void testServiceDataSource(WireMockRuntimeInfo wireMockRuntimeInfo) {
                     clientConfig,
                     MAPPER,
                     namespace,
-                    droveClient);
+                    droveClient, Collections.emptySet());
```

Comment:
We can move to Set.of() List.of() etc now that we've migrated to J17

Reply:
removed, not required